streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilderTest.java

```diff
@@ -747,14 +748,19 @@ public void shouldNotAllowNullTopicChooserWhenAddingSink() {
         assertThrows(NullPointerException.class, () -> builder.addSink("name", (TopicNameExtractor<Object, Object>) null, null, null, null));
     }
 
+    // TODO: check if this needs to be expanded
     @Test
     public void shouldNotAllowNullNameWhenAddingProcessor() {
-        assertThrows(NullPointerException.class, () -> builder.addProcessor(null, () -> null));
+        assertThrows(NullPointerException.class, () -> builder.addProcessor(null,
+                                                                            (ProcessorSupplier<Object, Object, Object, Object>) () -> null
+        ));
```

Comment:
The alignment is weird here. We either put the whole method call on one line (if it's short), or put all args on their own lines.

Eg, something like this:
```suggestion
        assertThrows(
            NullPointerException.class,
            () -> builder.addProcessor(
                    null,
                    (ProcessorSupplier<Object, Object, Object, Object>) () -> null
                )
        );
```